Gulpfile.js

```diff
@@ -0,0 +1,48 @@
+const gulp = require('gulp');
+const fs = require('fs');
+const connect = require('gulp-connect');
+const jade = require('gulp-jade');
+const rename = require('gulp-rename');
+
+gulp.task('connect', () => {
+  connect.server({
+    root: 'build',
+    port: 8000,
+    livereload: true,
+  });
+});
+
+gulp.task('jade', () => {
+  gulp.src('jade/*.jade')
+    .pipe(jade({
+      locals: JSON.parse(fs.readFileSync('./data-source.json', 'utf8')),
```

Comment:
1) The current Jade watcher allows the user to specify a particular json data source for each Jade template. It looks like this will use the same data source for every template, or will require the user to copy this 8-line gulp pipeline for each template-datasource pair. Can you make this a parameterized function? e.g. `watchAndRenderWithData(jadeTemplate, jsonDataFile)`

2) I intend to put data sources in a top-level `data/` directory, so that change is incoming.